image/templates/helm/shared/templates/02-scanner-v4-05-indexer-network-policy.yaml

```diff
@@ -14,6 +14,7 @@ spec:
     matchLabels:
       app: scanner-v4-indexer
   ingress:
+  {{- if ._rox.env.centralServices }}
```

Comment:
Just to make sure I get this right: indexer on central side shall be accessible from central and sensor while indexer when deployed on a secured cluster shall only be accessible from sensor.

Reply:
Correct. Note: we rely on Sensor -> Indexer for delegated scanning and OpenShift Internal Registry scanning purposes, so we need to always allow this.